app/controllers/ems_cloud_controller.rb

```diff
@@ -160,7 +160,7 @@ def ems_cloud_form_fields
                      :provider_id                     => @ems.provider_id ? @ems.provider_id : "",
                      :hostname                        => @ems.hostname,
                      :api_port                        => @ems.port,
-                     :api_version                     => @ems.api_version,
+                     :api_version                     => @ems.api_version ? @ems.api_version : "v2",
```

Comment:
> Although with the angular form, I think it is not supported to have default value per provider, since it doesn't do any requests when changing the provider type, the logic is in JS

Ugh that's horrible...that means all the business logic for new objects (*ANY* new object) has to be in Angular as well?  cc @matthewd @chessbyte  @blomquisg